frontend/src/components/Posts/post-plain.js

```diff
@@ -3,32 +3,38 @@ import { useState } from "react";
 import { author_api } from "../../api/post_display_api";
 
 export default function PlainPost(data) {
-    //Get Author data
-    const [author, setAuthor] = useState("");
-    author_api(data["post"]["author"], setAuthor);
+    //Get Author data -> author data is attached to data
     
     //Decide if shareable
     let shareable = (data["post"]["visibility"] === "PUBLIC" || data["post"]["visibility"] === "FRIENDS") ? true : false
 
+    const port = window.location.port ? `:${window.location.port}` : "";
+    const authorUrl = `//${window.location.hostname}${port}/user/${(data.post.author.id ?? "").split('/').pop()}`; // allows linking to the author who wrote the post
+
     return (
-        <div className="message">
-            <div className="from">
-                <img alt="author" src={author["profileImage"]}></img>
-            </div>
-            {/* Will need to handle other post types here, plain for now */}
-            <div className="content-container">
-                <h5>{data["post"]["title"]}</h5>
-                <div className="content">
-                    {data["post"]["content"]}
+        <div>
+            <div className="message">
+                <div className="from">
+                    <h6><a href={authorUrl}>{data.post.author.displayName}</a></h6>
```

Comment:
Would it be intuitive enough to use the image as a link to the author profile?

Reply:
I personally dont think it matters right now, but thats something we can likely add.

Reply:
Yeah, true, that's more of a detail we can worry about later